packages/sync-repo-settings/src/bin/commands/local-sync.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import yargs = require('yargs');
+import {logger} from 'gcf-utils';
+import {Octokit} from '@octokit/rest';
+import {SyncRepoSettings} from '../../sync-repo-settings';
+import * as yaml from 'js-yaml';
+import {RepoConfig} from '../../types';
+import {readFileSync} from 'fs';
+
+interface Args {
+  file: string;
+  'github-token': string;
```

Comment:
Let's just assume this is in `process.env.GITHUB_TOKEN` - that's what I've done elsewhere.